test/web-auth/cross-origin-authentication.test.js

```diff
@@ -267,7 +267,11 @@ describe('auth0.WebAuth.crossOriginAuthentication', function() {
               anotherOption: 'foobar'
             },
             function(err) {
-              expect(err).to.be.eql({ error: 'any_error', error_description: 'any error' });
+              expect(err).to.be.eql({
+                original: { error: 'any_error', error_description: 'any error' },
+                code: 'any_error',

```

Comment:
I know that it's different `any_error` to `any error`, but can you please change one of them to something more visibly different? i.e. `code: 'any_error', description: 'a long error description'`